### DISCUSSION TITLE
🌞 Add support for H1 inverters

### DISCUSSION THREAD

**Discussion by djansen1987** [djansen1987/SAJeSolar#15] [Announcements]
- Added support for H1 (store Device Power) inverters.

- Cleaned up the code.

If you have any issues let me know [https://github.com/djansen1987/SAJeSolar/issues](https://github.com/djansen1987/SAJeSolar/issues)

<hr /><em>This discussion was created from the release <a href='https://github.com/djansen1987/SAJeSolar/releases/tag/v1.2.5'>🌞 Add support for H1 inverters</a>.</em>

**Comment by rpalacios-ak** [comment]
Hi!

First of all, i want to thank you for your work. I really appreciate it.

I have a **SAJ H1-6K-S2** inverter with batteries and I would like to help you to improve the component. As you see, I've configured all the sensors in order to test which of them returns values, but there is a lot of "unknown" (_desconocido_ in Spanish):
![esolar2](https://user-images.githubusercontent.com/76500865/150516963-81cb359c-c349-4014-afb2-46ca5dcc43f5.png)

In fact, the **total** values are correct, but the only **instantaneous** value I get is "nowPower". No load power, self-consumption, export power, import power, battery charge / discharge or SOC. I can get all this information from https://fop.saj-electric.com/, so I think, maybe, parameters are just named differently for this inverter.

I could give you access to my eSolar portal if you want or helping you getting the data you required.

Regards.

**Comment by rpalacios-ak** [comment]
I've just checked that it's working perfectly. It was my mistake that I wasn't include **sensors: h1** into my configuration.yaml.

Anyway, let me know if you could need any help.

Thank you again for your time and your work.

Regards!
